en/assembly/quick_start_pixhack_v3x.md

```diff
@@ -0,0 +1,61 @@
+#Pixhack V2/V3/V3X wiring quick start
+
+The quick start guide shows how to power the Pixhack V2/V3 flight controller and connect its most important peripherals.
+
+
+![Pixhack v3x](../../assets/flight_controller/pixhack_v3x/pixhack_v3x.png)
+
+###Mount and Orient Controller
+
+Pixhack V2/V3/V3X should be mounted on the frame using vibration-damping foam pads (included in the kit). It should be positioned as close to your vehicle’s center of gravity as possible, oriented top-side up with the arrow pointing towards the front of the vehicle.
+
+<img src="../../assets/flight_controller/pixhack_v3x/pixhack_v3_vehicle_feont.jpg" align="center"/>
+
+> **Note** If the controller cannot be mounted in the
+  recommended/default orientation (e.g. due to space constraints) you will
+  need to configure the autopilot software with the orientation that you
+  actually used: [Flight Controller Orientation](../config/flight_controller_orientation.md).
+  
+###Wiring Overview
+
+The image below shows how to connect the most important sensors and peripherals. We'll go through each of these in detail in the following sections.
+
+<img src="../../assets/flight_controller/pixhack_v3x/pixhack_v3_wiring_overvie.jpg" align="center"/>
```

Comment:
File name appears to have typo error: _overvie.jpg (ie you probably mean "overview"). It does match file.